orte/mca/rmaps/base/rmaps_base_binding.c

```diff
@@ -236,7 +239,10 @@ static int bind_generic(orte_job_t *jdata,
                 }
             }
             /* bind the proc here */
-            hwloc_bitmap_or(totalcpuset, totalcpuset, trg_obj->cpuset);
+            // was: hwloc_bitmap_or(totalcpuset, totalcpuset, trg_obj->cpuset);
+            cpus = opal_hwloc_base_get_available_cpus(node->topology->topo, trg_obj);
```

Comment:
This `opal_hwloc_base_get_available_cpus` looks like the offending call.

Reply:
Yeah - it comes from a much larger patch that mark did that will need some eyeballing.